frontend/plugins/sales_ui/src/modules/pos/pos-detail/components/posDetail.tsx

```
@@ -1,303 +1,241 @@
       </PosEditTabContent>
 
       <PosEditTabContent value="sync">
         <SyncCardForm posDetail={posDetail} />
       </PosEditTabContent>
+
+      {posCategory === 'restaurant' && (
+        <PosEditTabContent value="slot">
+          <POSSlotsManager
+            posId={posDetail._id}
+            initialNodes={[]}
+            onNodesChange={() => {}}
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)